include/mbgl/storage/default_file_source.hpp

```diff
@@ -195,6 +190,11 @@ class DefaultFileSource : public FileSource {
      */
     void packDatabase(std::function<void(std::exception_ptr)> callback);
```

Comment:
Noting that this is a particularly confusing API name since we call offline regions "offline packs" on iOS 🤕 but I understand that this API name predates this PR.

Would we accept an update to the documentation like the following to clarify what the API does?

> Packs the existing database file into the minimum disk space required by invoking VACUUM on SQLite. If resources have been removed from the database, this operation would shrink the database size to reflect the remaining resources.

> When the operation is complete or encounters an error, the given callback will be executed on the database thread; it is the responsibility of the SDK bindings to re-execute a user-provided callback on the main thread.

(no change to the second paragraph)

Reply:
note: if this API does more than run vacuum, let's include in the first sentence ^